src/yb/rocksdb/util/thread_posix.h

```diff
@@ -102,6 +102,7 @@ class ThreadPool {
     void (*function)(void*);
     void* tag;
     void (*unschedFunction)(void*);
+    yb::util::AUHAuxInfo aux_info;
```

Comment:
Why do we need this?

Reply:
When we are scheduling bg events we also need to store the aux info which is in stored in the current threadlocal, otherwise we are going to lose it as this bg event is going to be picked up by some other thread.